src/pet.js

```diff
@@ -0,0 +1,125 @@
+//magic numbers
```

Comment:
Remove comments like this. Comments should describe why you're doing something not what you're doing